alfalfa_worker/lib/__init__.py

```diff
@@ -1,3 +1,4 @@
-from .precheck_argus import *  # noqa
-from .make_gzip_file import *  # noqa
-from .tagutils import *  # noqa
+# TODO: these should be imported into the files that need them
```

Comment:
I try not to initialize a bunch of files. I updated the includes in the files that needed them.